tap.js

```diff
@@ -28,28 +29,52 @@
         this.el.addEventListener('mousedown', this, false);
     }
 
-    Tap.prototype.start = function(e) {
+    function fixupMouse( event ) {
+        event = event || window.event;
+        var e = { event: event,
+            target: 'target' in event ? event.target : event.srcElement,
+            which: 'which' in event ? event.which : // otherwise MSIE<8
+                 event.button === 1 ? 1 :   // left
+                 event.button === 2 ? 3 :   // right
+                 event.button === 4 ? 2 : 1,// center
+            buttons: 0,
+            button: 0,
+            x: 'x' in event ? event.x : event.clientX,
+            y: 'y' in event ? event.y : event.clientY
+        };
+        e.button = e.which - 1;
+        e.buttons = 'buttons' in event ? event.buttons :
+                 e.which === 1 ? 1 :   // left
+                 e.which === 3 ? 2 :   // right
+                 e.which === 2 ? 4 : 1;// center
+        return e;
```

Comment:
Instead of modifying the event object and returning it, could we just simplify this function to only detect a left click and return a Boolean value instead?